indico/modules/categories/templates/display/overview/base.html

```diff
@@ -0,0 +1,123 @@
+{% extends 'layout/management_page.html' %}
+
+{% block side_menu %}
+    <div class="category-overview-side-menu">
+        <form action="{{ url_for('.overview', category) }}" method="GET" id="category-overview-options">
+            <span id="calendar-container"></span>
+            <input type="hidden" id="date" name="date" value="{{ start_dt|format_date(format="YYYY-MM-dd") }}">
+            <h3>{% trans %}Display options{% endtrans %}</h3>
+            <table cellpadding="0" cellspacing="0" style="width:100%">
+                <tr>
+                    <td>{% trans %}Period:{% endtrans %}</td>
+                    <td>
+                        <select name="period">
+                            <option value="day" {%- if period == 'day' %} selected{% endif %}>
```

Comment:
I wouldn't even bother about the whitespace and simply always use the one from outside. looks better IMO, even if the generated HTML is uglier:

    <option value="day" {% if period == 'day'%} selected{% endif %}>